guide/getting-started/hardware.md

```diff
@@ -145,6 +147,26 @@ Bitcoin ATMs are usually bound to local money transmission laws and regulations,
 
 You can find local ATMs using services like [coinatmradar](https://coinatmradar.com/).
 
+## Point of sale
+
+People looking to accept bitcoin payments, especially when using the lightning network, sometimes use dedicated point of sale (POS) hardware terminals when selling goods and services.
+
+{% include picture.html
+   image = "/assets/images/guide/getting-started/hardware/point-of-sale.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/point-of-sale@2x.jpg"
+   mobile = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile.jpg"
+   mobileRetina = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile@2x.jpg"
+   alt-text = "Examples of point of sale hardware devices."
+   width = 800
+   height = 400
+%}
+
+Dedicated bitcoin POS hardware is used to request and receive bitcoin payments in-person. These act very similar to traditional POS systems but present to payees a bitcoin on-chain, lightning, or [unified](https://bitcoinqr.dev/) payment request. Payment requests can also be shared and paid over NFC.
```

Comment:
Not sure how important it is to point out the different types of payment requests here...

For NFC, maybe add "if the POS hardware and user devices support it"?